### ISSUE TITLE
Password appears in username spot and replaces text on log in button

### ISSUE THREAD

**Issue by baimeeker** [HabitRPG/habitica#1464]
On the regular habit site, I tried to log in and saw something a bit weird. I had apparently saved my habit username and password in my browser, and instead of putting the username and password in the login positions, my password was inserted as plaintext in the username area.  Strangely, it also replaced the text on the login button. There were stars in the password area, which I assume was also my password.

I think the password was saved a while back when I was using chrome on my computer before I disabled password saving.  Password saving was disabled on my mobile app, and I couldn't see that habit had a password listing until I re enabled it (not showing what passwords are saved when you stop actively saving them is an error on Chrome's side...)

Browser: chrome for mobile (on an iPad), version 28.0.1500.16
Username: baimeeker
No java errors

**Comment by lefnire** [comment]
could this have to do with #1460 ?

**Comment by lefnire** [comment]
can you git it another go? I removed some dynamic form-field calculation which I think may have fixed the issue

**Comment by baimeeker** [comment]
I really REALLY wish I could, but none of my chrome browsers currently recognize that I am logging into a website when I log into Habit, so they don't ask if I want to save the password.  I don't usually save passwords in browsers, so I deleted the original password listing right after I reported this problem.  (I did sign into amazon and it asked if I wanted to save my password... so it isn't the browser.)  Sorry!
